### DISCUSSION TITLE
Setting up simple tailwind installation no Framewrok

### DISCUSSION THREAD

**Discussion by cubecleveland** [tailwindlabs/tailwindcss#5021] [Help]
Hello,

I'm building a simple one page website, no framework.
I am not using any bundler or builder but want to use tailwind.
I manged to install tialwind via npx tailwindcss -o tailwind.css and i included tailwind.css in my html file.

I do however get the full tailwind.css file.

How do i optimize that for production removing all the styles i dont need.
I was trying to add postCss but how to do it vanilla style not really documented.
I really appreciate your help

**Comment by dance2die** [comment]
Looks like the `--purge` option for the Tailwind CLI hasn't made it into the documentation yet.  

Check out the blog, [Tailwind CSS v2.2-All-new high-performance Tailwind CLI.
](https://blog.tailwindcss.com/tailwindcss-2-2#all-new-high-performance-tailwind-cli)
You can purge as shown below.

```bash
npx tailwindcss -o dist/tailwind.css --watch --jit --purge="./src/**/*.html"
```

It's a v2.2 feature so make sure you have v2.2.x
